mlir/include/mlir/Dialect/LLVMIR/GENXOps.td

```diff
@@ -39,6 +39,9 @@ def GENX_Dialect : Dialect {
   let hasOperationAttrVerify = 1;
 
   let extraClassDeclaration = [{
+    /// Get the name of the attribute used to annotate external kernel
+    /// functions.
+    static StringRef getKernelFuncAttrName() { return "opencl.kernels"; }
```

Comment:
Can we name the attribute "genx.kernel" (similar to "nvvm.kernel")

Reply:
Good idea!